base/multidimensional.jl

```diff
@@ -1176,7 +1176,7 @@ copyto!(::AbstractArray, ::CartesianIndices, ::AbstractArray, ::CartesianIndices
 # circshift!
 circshift!(dest::AbstractArray, src, ::Tuple{}) = copyto!(dest, src)
 """
-    circshift!(dest, src, shifts)
+    circshift!(dest::AbstractArray, src, shifts)
```

Comment:
```suggestion
    circshift!(dest, src, shifts)
```
While the specific implementation here may be for an `AbstractArray`, in principle this may be implemented for any mutable destination.